pages/linux/pdftohtml.md

```diff
@@ -0,0 +1,20 @@
+# pdftohtml
+
+> Convert PDF files into HTML, XML and PNG images.
+> More information: <https://manned.org/pdftohtml.1>.
```

Comment:
```suggestion
> More information: <https://manned.org/pdftohtml>.
```

Reply:
what's the difference?

Reply:
> what's the difference? 

.1 isn't necessary in the URL if there isn't another command/repackage with the same name. The suggestion is a minor cosmetic fix.

Reply:
Oh, got it.